### PR TITLE
Plat 33 trigger lambda on cf events

### PR DESCRIPTION
**Pull request by perception30**

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -13,9 +13,29 @@ provider:
...
-          path: index
+      - cloudwatchEvent:
+          event:
+            source:
+              - "aws.cloudformation"
+            detail:
+              eventSource:
+                - "cloudformation.amazonaws.com"
+              eventName:
+                - "CreateStack"
```
Comment (jordan-simonovski) [inline serverless.yml]: Are there events like "create_complete" or "update_complete" that we could use. The problem being that if the stack fails to provision, we would also need to remove it from the Dynamo table.

File: tsconfig.json [hunk 2]
```diff
@@ -4,10 +4,10 @@
...
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "sourceMap": true,
+    "sourceMap": false,
     "target": "es2017",
     "outDir": "lib",
     "allowSyntheticDefaultImports": true
   },
-  "include": ["typings/**/*.d.ts", "src/**/*.ts", "src/**/*.tsx"]
+  "exclude": ["src/**/*.spec.ts"]
```
Comment (jordan-simonovski) [inline tsconfig.json]: /nit can we change .spec to .test?

Comment (perception30) [reply tsconfig.json]: Yeah, Just did that.

File: serverless.yml (hunk 1 above)
Comment (jordan-simonovski) [reply serverless.yml]: Nice one! I can approve this now, but we'll need a task to set up the state machine for polling the stack status

**Pull request merged**